project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 object Dependencies {
 
   object Version {
-    val scala212 = "2.12.12"
+    val scala212 = "2.12.13"
```

Comment:
We can't migrate to 2.12.13 yey because scoverage doesn't support it yet (see https://github.com/scoverage/sbt-scoverage/issues/321). This is also what's making the build fail.

Is it required for any of the modules?

Reply:
No, it is not required. I will downgrade.